bin/check-rabbitmq-consumers.rb

```diff
@@ -85,74 +56,56 @@ class CheckRabbitMQConsumers < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 2
 
-  option :ini,
-         description: 'Configuration ini file',
-         short: '-i',
-         long: '--ini VALUE'
-
-  def rabbit
-    begin
-      if config[:ini]
-        ini = IniFile.load(config[:ini])
-        section = ini['auth']
-        username = section['username']
-        password = section['password']
-      else
-        username = config[:username]
-        password = config[:password]
-      end
-
-      connection = CarrotTop.new(
-        host: config[:host],
-        port: config[:port],
-        user: username,
-        password: password,
-        ssl: config[:ssl]
-      )
-    rescue StandardError
-      warning 'could not connect to rabbitmq'
-    end
-    connection
+  def queue_list_builder(input)
```

Comment:
rather than redefining this in each script lets throw it into the shared libraries.